hack/local-cmo.sh

```diff
@@ -100,6 +148,12 @@ main(){
   local -a images
   while read -r img; do images+=( "$img" ); done < <(images_from_deployment)
 
+  if [ $cmo_login == 'true' ]; then
+    echo "loggin as cmo sa"
+    echo "run: oc login -u system:admin when finished in order to continue as admin"
+    login_as_cmo
+  fi
```

Comment:
```suggestion
  if  $cmo_login; then
    echo "logging as cmo sa"
    login_as_cmo
    echo "run: oc login -u system:admin when finished in order to continue as admin"
  fi
```

Reply:
I think we can improve the script by switching back to the old use when the script exits `trap restore_login EXIT`